app/styles/_app-nav-primary.scss

```diff
@@ -0,0 +1,72 @@
+.app-nav-primary {
+  background-color: $bGColorBase;
+  border-right: 2px solid $colorSubtle;
+  bottom: 0;
+  box-shadow: $baseShadow;
+  left: -250px;
+  overflow-y: scroll;
+  padding: 3rem 1rem 1rem;
+  position: fixed;
+  top: 0;
+  transition: left $animationSpeedBase;
```

Comment:
If you transition a `transform: translate-x`, the animation will take place off the main thread.

Reply:
Today I learned! This feels like I was thinking oldschool CSS so thanks for a more modern way to do this.